dotcom-rendering/src/web/components/Newsletters/NewsletterFrequency.tsx

```diff
@@ -0,0 +1,35 @@
+import { css } from '@emotion/react';
+import { space, textSans } from '@guardian/source-foundations';
+import { SvgClock } from '@guardian/source-react-components';
+
+type Props = {
+	frequency: string;
+};
+
+const mainStyles = css`
+	display: flex;
+	margin-top: ${space[1]}px;
+`;
+
+const spanStyles = css`
+	margin-left: ${space[1]}px;
+	${textSans.xsmall()}
+	strong {
+		${textSans.xsmall({ fontWeight: 'bold' })}
+	}
+`;
+
+export const NewsletterFrequency: React.FC<Props> = ({ frequency }) => {
```

Comment:
Is this component used anywhere? I can only see it being imported into the story